packages/app-project/stores/Project.js

```diff
@@ -0,0 +1,44 @@
+import { flow, types } from 'mobx-state-tree'
+import _ from 'lodash'
+import axios from 'axios'
+
+function transformResponse (response) {
+  return _.get(response, 'data.projects[0]')
+}
+
+function fetchProjectData (slug) {
+  return axios({
+    url: 'https://www.zooniverse.org/api/projects',
+    params: { slug },
+    headers: {
+      'Accept': 'application/vnd.api+json; version=1',
+      'Content-Type': 'application/json'
+    }
+  }).then(transformResponse)
+}
+
+const Project = types
+  .model('Project', {
+    isLoading: types.boolean,
+    data: types.optional(types.frozen, {})
+  })
+  .actions(self => ({
+    fetch: flow(function * fetch (slug) {
+      try {
+        self.isLoading = true
+        self.data = yield fetchProjectData(slug)
+      } catch (error) {
+        console.error('Failed to fetch project', error)
+      }
+      self.isLoading = false
+    })
+  }))
+
+const defaultProjectState = {
+  isLoading: false
```

Comment:
I may be misunderstanding MST, but I'm thinking that this could be a model instance like:
```
const defaultProjectState = Project.create({
  isLoading: false
})
```

This is discussed here: https://github.com/mobxjs/mobx-state-tree/#composing-trees

ETA: I see this is being done in the tests. Would there be any use of exporting an instance directly from the store?

Reply:
I think this was me being reluctant to adopt the MST way of using types.optional() to populate initial values - I'm going to change this, because it's just the way to do it.